types/sharepoint/index.d.ts

```diff
@@ -1,10 +1,11 @@
-// Type definitions for Microsoft SharePoint: 2016.1
+// Type definitions for Microsoft SharePoint: 2016.2
```

Comment:
you don't need to version bump, the typescript bot will do it automatically for you in this context

Reply:
sure :)